webcompat/static/js/lib/issues.js

```diff
@@ -234,7 +234,7 @@ issues.ImageUploadView = Backbone.View.extend({
     var DELIMITER = '\n\n';
     var textarea = $('.js-Comment-text');
     var textareaVal = textarea.val();
-    var imageURL = _.template('![Screenshot of the site issue](<%= url %>)');
+    var imageURL = _.template('[![Screenshot of the site issue](<%= url %>)](<%= url %>)');
```

Comment:
yes. it's not fixed. I tried to compute the thumb url in the same way we do in bugform.js but then I realized it's a duplication of code. What is the best way to create a function which is usable in both code, like we do import in python.